favorites.py

```diff
@@ -118,8 +130,12 @@ def check_aliases(cls, favs):
                 data['groups'][group] = [{"file": filename, "alias": basename(filename)}
                                          for filename in data['groups'][group]]
             data["version"] = 2
-            with open(favs, "w") as file:
-                json.dump(data, file, sort_keys=True, indent=4, separators=(',', ': '))
+
+            # in global file only, entry to store toggle project state
+            if basename(favs) == 'favorite_files_list.json' and 'project_mode' not in data:
+                data['project_mode'] = False
+
+            cls.fav_file(favs, data=data, write=True)
 

```

Comment:
same here, it also adds the 'project_mode' key if not present, where 'toggle for project' state is stored.